commons/all/src/main/java/org/infinispan/commons/marshall/ProtoStreamTypeIds.java

```diff
@@ -53,6 +53,7 @@
    int META_PARAMS_INTERNAL_METADATA = CORE_LOWER_BOUND + 3;
    int REMOTE_METADATA = CORE_LOWER_BOUND + 4;
    int UUID = CORE_LOWER_BOUND + 5;
+   int PRIVATE_METADATA = UUID + 1;
```

Comment:
We probably want this in the `0..127` range as it is included in so many commands.

Reply:
done! 
is there a special case for ids < 128?

Reply:
Ids < 128 only require a single byte to marshall. But obviously we only have a limited number of messages that can be in this range, so we should save this range for protobuf messages which are sent frequently